content/docs/command-reference/run.md

```diff
@@ -268,6 +268,17 @@ $ dvc run -n second_stage './another_script.sh $MYENVVAR'
 - `--desc <text>` - user description of the stage (optional). This doesn't  
   affect any DVC operations.
 
+- `--live <path>` - specify the directory `path` for
+  [Dvclive](/doc/dvclive/dvclive-with-dvc) to write logs in. Saved in the `live`
+  field of `dvc.yaml`.
```

Comment:
- Unrelated Q: Is it worth maintaining `dvc run` going fwd or should we just redirect to `stage add`? I'm not 100% sure they're exactly the same at this point?